Tools/AP_Periph/can.cpp

```diff
@@ -1708,7 +1715,14 @@ void AP_Periph_FW::esc_telem_update()
         if (esc_telem.get_raw_rpm(i, rpm)) {
             pkt.rpm = rpm;
         }
-        pkt.power_rating_pct = 0;
+
+#if AP_EXTENDED_ESC_TELEM_ENABLED
```

Comment:
Technically this is not extended, just new to this PR? Should it really be classified as such?

Reply:
Not new in this PR. Its not in the DroneCAN extended message, but it is a thing we only get over DroneCAN none of the other ESC protocols report it. The extended telem define and log seemed like a better fit than trying to squeeze it into the existing log where all the other protocol would have to pay the cost too.